e107_core/shortcodes/batch/login_shortcodes.php

```diff
@@ -151,15 +151,19 @@ function sc_login_table_rememberme($parm=null)
 
 	}
 	
+	/* example {LOGIN_TABLE_SUBMIT} */
+	/* example {LOGIN_TABLE_SUBMIT=large}  old way */  
+	/* example {LOGIN_TABLE_SUBMIT: class=btn btn-primary btn-large btn-lg pull-right} */  
 	function sc_login_table_submit($parm="") //FIXME use $frm
 	{
-		if(empty($this->userReg))
+    $class = ($parm == 'large') ? "btn-large btn-lg" : "";
+    if(empty($this->pref['user_reg']))
 		{
 			return null;
 		}
-
-		$class = ($parm == 'large') ? "btn-large btn-lg" : "";
-		return "<input class='btn btn-primary ".$class." button' type='submit' name='userlogin' value=\"".LAN_LOGIN_9."\" />";
+ 
```

Comment:
I would prefer to remove that old way totally, not new shorcode. So no large parm.